frontend/.gitignore

```diff
@@ -0,0 +1,23 @@
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# next.js
+/.next/
+/out/
+
+# production
+/build
+
+# misc
+.DS_Store
+.env*
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
please add package-lock.json also in `.gitignore`

Reply:
I guess it should not be added to .gitignore. It's similar, as done in server folder.

Reply:
it should be removed from both server and frontend directories....i'll check that, fine for now